spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :request do
+  describe 'GET #index' do
+    before { get user_posts_path(user_id: 1) }
+
+    it 'returns a successful response' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the index template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      expect(response.body).to include('Here is a list of posts for given user')
+    end
+  end
+
```

Comment:
- [ ] As you can see in the image below, two of your tests are failing, I will encourage to run all tests and ensure they are passing before making another submission
![image](https://github.com/firstpj/BLOG-appllication/assets/86200356/677df967-8867-4070-b854-99262013f3f5)